### PR TITLE
Added oral rehydration therapy drink and recipes to craft it.

### PR DESCRIPTION
**Pull request by Inglonias**
#### Summary
```SUMMARY: Content "Added rehydration drink based on oral rehydration therapy"```

#### Purpose of change
Added a rehydration drink. This is take two of #37256 

#### Describe the solution
Two liters of a rehydration drink can be made with a tablespoon of salt and ten tablespoons of sugar. Because the smallest unit of salt in this game is 5g, 2L is the minimum that can be made.

#### Describe alternatives you've considered
Originally this recipe was going to just make a sports drink.

#### Testing
Crafted the recipe. It seems like it works.

#### Additional context
image

### CODE REVIEW

**Comment by Inglonias** [comment]
It looks like a bunch of comestible tests are failing, and I'm not sure why.

File: data/json/items/comestibles/drink.json [hunk 1]
```diff
@@ -847,6 +847,25 @@
...
+    "symbol": "~",
+    "quench": 60,
+    "calories": 12,
+    "description": "A basic oral rehydration therapy drink.  It will rehydrate you faster than water, but it tastes awful.",
+    "price": 55,
+    "material": "water",
+    "volume": "250 ml",
+    "looks_like": "sports_drink",
+    "phase": "liquid",
+    "fun": -5
```
[the remark this answers is not in the document]
Comment (Inglonias) [reply data/json/items/comestibles/drink.json]: Setting joy to -1 makes this no different from standard sports drink.

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -5635,6 +5635,19 @@
...
+    "result": "rehydration_drink",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_DRINKS",
+    "book_learn": [ [ "textbook_chemistry", 1 ], [ "manual_first_aid", 1 ] ],
+    "skill_used": "cooking",
+    "time": "3 m",
+    "charges": 8,
+    "autolearn": false,
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "components": [ [ [ "sugar", 10 ] ], [ [ "salt", 1 ] ], [ [ "water_clean", 8 ] ] ]
```
[the remark this answers is not in the document]
Comment (Inglonias) [reply data/json/recipes/recipe_food.json]: See recipe posted above. No soda bicarbonate required. Other sources indicate similar things. As you said in the other pr, this is not sports drink

[the remark this answers is not in the document]
Comment (I-am-Erk) [reply data/json/recipes/recipe_food.json]: The recipe above is a curio sidebar from a nonmedical book, not a medical recipe for ORT. Some kind of alkali is included to counter metabolic acidosis from diarrhoea or vomiting, and citrate or bicarb would be recommended in the books we're describing.

* [medscape article](https://www.medscape.com/answers/801012-76356/what-is-the-alternative-to-commercially-prepared-oral-rehydration-solution-ors-for-the-treatment-of-pediatric-dehydration)
* [one of the ancient papers describing it](https://adc.bmj.com/content/archdischild/59/11/1072.full.pdf)

File: data/json/items/comestibles/drink.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (I-am-Erk) [reply data/json/items/comestibles/drink.json]: -2 would work then. I forgot sports drink has a negative value when warm.

File: data/json/recipes/recipe_food.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Inglonias) [reply data/json/recipes/recipe_food.json]: You're correct in that regard, however we're not treating diarrhea in-game. We're simply attempting to hydrate more efficiently than just water. That being said, the places where you'd get this recipe wouldn't know that about us, and would ask for bicarbonate. I'm willing to concede the point here, but agree that adding baking soda to the game for just this recipe would be out of scope for now.

File: data/json/items/comestibles/drink.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Inglonias) [reply data/json/items/comestibles/drink.json]: Joy set to -2

**Pull request merged**